torch/_inductor/codegen/cpp_gemm_template.py

```diff
@@ -490,7 +490,7 @@ def normalize_shapes(inputs, layout_or_out):
                 if trans_w:
                     assert isinstance(W, torch.Tensor)
                     W = W.transpose(0, 1)
-            if B is not None:
+            if has_bias and B is not None:
```

Comment:
Why do we need this change?

Reply:
`B` is `inputs[2]`. In case of WoQ GEMM, that's the scale. The current code in the main branch wouldn't cause any issue in the WoQ GEMM case, though, since scale is expanded in the epilogue for broadcasting, anyway.